src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -54,6 +54,57 @@
     short-summary: Manage certificates.
 """
 
+helps['keyvault certificate get-default-policy'] = """
+    type: command
+    short-summary: Get a default policy for a self-signed certificate
+    long-summary: >
+        This default policy can be used in conjunction with `az keyvault create` to create a self-signed certificate.
```

Comment:
Closer, but I would argue that for simple types we should include a sample value (string, int, datetime, etc).  For things like lifetimeActions (which I believe there are only two types) include an example of each. The idea would be that someone could use this as a basis for editing and not have to go look things up somewhere (like the tests!) where it may or may not exist.